File: src/test/java/com/everyonewaiter/domain/notification/AlimTalkMessageTest.java
package com.everyonewaiter.domain.notification;

import static com.everyonewaiter.domain.notification.AlimTalkWeblinkButtonTemplate.MENU_PREVIEW;
import static com.everyonewaiter.domain.notification.NotificationFixture.createAlimTalkMessage;
import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class AlimTalkMessageTest {

  @Test
  void addButton() {
    AlimTalkMessage alimTalkMessage = createAlimTalkMessage();

    alimTalkMessage.addButton(MENU_PREVIEW, "https://everyonewaiter.com", 1L);

    assertThat(alimTalkMessage.getButtons()).hasSize(1);
  }

}

File: src/test/java/com/everyonewaiter/domain/notification/AlimTalkMessageTest.kt
package com.everyonewaiter.domain.notification

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class AlimTalkMessageTest {

  @Test
  fun `알림톡 버튼 추가`() {
    val message = createAlimTalkMessage()

    message.addButton(AlimTalkWeblinkButtonTemplate.MENU_PREVIEW, 1L)

    assertThat(message.buttons).hasSize(1)
  }
}
